pkg/mimirtool/commands/alerts.go

```diff
@@ -74,8 +74,8 @@ func (a *AlertmanagerCommand) Register(app *kingpin.Application, envVars EnvVarN
 
 	alertCmd.Command("delete", "Delete the Alertmanager configuration that is currently in the Grafana Mimir Alertmanager.").Action(a.deleteConfig)
 
-	loadalertCmd := alertCmd.Command("load", "Load a set of rules to a designated Grafana Mimir endpoint").Action(a.loadConfig)
-	loadalertCmd.Arg("config", "alertmanager configuration to load").Required().StringVar(&a.AlertmanagerConfigFile)
+	loadalertCmd := alertCmd.Command("load", "Load Alertmanager tenant configuration and template files to Grafana Mimir.").Action(a.loadConfig)
```

Comment:
```suggestion
	loadalertCmd := alertCmd.Command("load", "Load Alertmanager tenant configuration and template files into Grafana Mimir.").Action(a.loadConfig)
```

Reply:
(Double checking this preposition with fellow writers, and can catch this in another PR if you want to leave it or change it.)